docs/Sprint-Review/sprint-48-summary.md

```diff
@@ -0,0 +1,35 @@
+# Sprint 48 Summary
+5/25/22-6/07/22
+
+Velocity: 8
+## Sprint Goals
+Design the pilot user onboarding content in preparation for the v2.0 release. Additionally, mature the production environment and establish the file transfer mechanism for the pilot program.
```

Comment:
@lfrohlich Although this is similar to the sprint 49 goal, since the S/W that is being worked is the same, it is different than our sprint 49 goal which will implement the Knowledge Center.